### ISSUE TITLE
Support CTRL+CLICK a file/folder

### ISSUE THREAD

**Issue by Rickkwa** [mintty/wsltty#115]
Using Cygwin which also uses Mintty, if you run `ls` and then CTRL+click a file/folder that it listed, Mintty will open that file/folder. For example, if I CTRL+click on a folder listed in the `ls` output, it will open that folder in a new Explorer window. If I CTRL+click on a text file, it will open that file with my default text editor.

With wsltty, when I CTRL+click on a file/folder, I get the error:

```
The system cannot find the file specified.
C:\Users\<username>\<filename>
```

It seems to look for the file in `C:\Users\<username>` which is not where the file is actually stored.

**Comment by mintty** [comment]
Ctrl+click filename mapping in WSL has been improved recently (which version do you run?).
However, if it's a *relative* path name, mintty cannot detect the current working directory of WSL, so that cannot work. For a workaround, you can use the escape sequence to explicit set the current directory, e.g. embedded in your prompt. See the wiki page Tips about this.

**Comment by Rickkwa** [comment]
I was using mintty 2.8.5; wsltty 1.8.5.2 at the time of the original post. I just upgraded to mintty 2.9.0 and wsltty 1.9.0.2 and am seeing the same issue.

If I do something like `echo /some/absolute/path` and ctrl+click on the output, it works. However, I'm curious as to why cygwin doesn't have this issue. And it seems cygwin can handle relative paths? I.e. I ctrl+click on `../test` and it can resolve that.

**Comment by mintty** [comment]
Cygwin can detect the current foreground process and its current working directory.
This does not work over the boundaries of wslbridge and WSL.

**Comment by sarim** [comment]
In WSL2 this got broken. If I inform PWD using escape sequence and then try to Ctrl+Click on a filename, wsltty/mintty tries to map it to wsl1 style path inside APPDATA, but in wsl2 the files are no longer there, as its inside the ext4 image.
Any tips on how to make mintty use the `\\wsl$\Ubuntu-18.04\` style path?

**Comment by mintty** [comment]
I'll take that as a feature request but I'd like more information about that new mapping, i.e. a link to an authoritative description please.

**Comment by sarim** [comment]
@mintty I couldn't find exact article describing it but this is not new. It works for both wsl1&2 and this is the official and recommended way to access linux filesystem from windows. 
Here is a article from Microsoft https://devblogs.microsoft.com/commandline/do-not-change-linux-files-using-windows-apps-and-tools/

![image](https://user-images.githubusercontent.com/1235888/86947024-0c4a1880-c16d-11ea-817c-f9650ed1fbe1.png)

**Comment by mintty** [comment]
The `\\wsl$\` syntax works for some WSL distributions but not for all. Mintty will check the current path (fixed, by the way) first and if it does not exist it will fallback to the \\wsl$ path.

**Comment by mintty** [comment]
Released 3.3.0.
